critiquebrainz/frontend/forms/review.py

```diff
@@ -19,14 +19,32 @@ def __call__(self, form, field):
         if l < self.min or self.max != -1 and l > self.max:
             raise ValidationError(self.message)
 
+other_languages_codes = supported_languages
+frequently_used_languages_codes = list(get_top_languages())
+for language in frequently_used_languages_codes:
+    other_languages_codes.remove(language)
+other_languages = []
+frequently_used_languages = []
+for language_code in other_languages_codes:
+    try:
+        other_languages.append((language_code, Locale(language_code).language_name))
+    except UnknownLocaleError:
+        other_languages.append((language_code, pycountry.languages.get(iso639_1_code=language_code).name))
+for language_code in frequently_used_languages_codes:
+    try:
+        frequently_used_languages.append((language_code, Locale(language_code).language_name))
+    except UnknownLocaleError:
+        frequently_used_languages.append((language_code, pycountry.languages.get(iso639_1_code=language_code).name))
+
 
 # Loading supported languages
 languages = []
-for language_code in supported_languages:
-    try:
-        languages.append((language_code, Locale(language_code).language_name))
-    except UnknownLocaleError:
-        languages.append((language_code, pycountry.languages.get(iso639_1_code=language_code).name))
+languages.append(("","Frequently Used Languages"))
```

Comment:
I fixed some things on your code to make it run. But the `frequently used language` thing should not be selectable and must only be a label. But it is selectable here in my setup:
`Frequently Used Languages` is selectable. It should be just a different-looking label. Similar should be the label `Other Languages`
![12](https://user-images.githubusercontent.com/12391372/34093492-867ef2c4-e3ee-11e7-83ff-fc472e5f9d5b.png)